scripts/build-ios-arm64.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
```

Comment:
This change is unrelated to the PR, please leave it out

Reply:
Got it